gtsfm/configs/sift_front_end_astrovision.yaml

```diff
@@ -34,8 +34,8 @@ SceneOptimizer:
 
     inlier_support_processor:
       _target_: gtsfm.two_view_estimator.InlierSupportProcessor
-      min_num_inliers_est_model: 15
-      min_inlier_ratio_est_model: 0.1
+      min_num_inliers_est_model: 1500
```

Comment:
curious about the change from 15 to 1500? what was the performance difference?

Reply:
Can you post the dashboard for this change?

Reply:
This was tuning i did for a specific sequence, I will changed back

Reply:
fixed